config-agent.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Starknet-Agent-Kit",
```

Comment:
So this will be the default config loaded if nothing is set? Could you move this under a ./config/agents/default-agent.json folder so that other preset config can be set there